store/registry.go

```diff
@@ -132,7 +133,34 @@ func (rs *registryStore) GetAgentsIfChanged(scanner string) ([]*config.AgentConf
 		rs.lastCompletedVersion = hash.Hash // remember next time so we don't retry the same list
 	}
 
-	return loadedBots, true, nil
+	// populate bots with redundancy
+	var allBots []*config.AgentConfig
+	for _, bot := range loadedBots {
+		if bot.Redundancy == 0 {
+			bot.Redundancy = 1
+		}
+
+		for i := uint(0); i < bot.Redundancy; i++ {
```

Comment:
should this be looping over shards instead (redundancy is across nodes, shard is within one)?

Reply:
> redundancy is across nodes, shard is within one
agreeing with this terminology.